packages/compass-import-export/src/components/export-json-format-options.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+  Accordion,
+  Link,
+  Radio,
+  RadioGroup,
+  css,
+  spacing,
+} from '@mongodb-js/compass-components';
+
+import type { ExportJSONFormat } from '../export/export-json';
+
+const radioGroupStyles = css({
+  margin: `${spacing[3]}px 0`,
+});
+
+function JSONFileTypeOptions({
+  jsonFormat,
+  setJSONFormatVariant,
+}: {
+  jsonFormat: ExportJSONFormat;
+  setJSONFormatVariant: (jsonFormatVariant: ExportJSONFormat) => void;
+}) {
+  return (
+    <Accordion
+      text="Advanced JSON Format"
+      data-testid="export-advanced-json-format"
+    >
+      <RadioGroup
+        className={radioGroupStyles}
+        data-testid="export-json-format-options"
+        onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+          setJSONFormatVariant(event.target.value as ExportJSONFormat)
+        }
+      >
+        <Radio
+          value="default"
+          checked={jsonFormat === 'default'}
+          description='Example:  { "fortyTwo": 42, "oneHalf": 0.5, "bignumber": { "$numberLong": "5000000000" } }'
+        >
+          Default Extended JSON
+        </Radio>
+        <Radio
+          value="relaxed"
```

Comment:
Would be nice if we had a warning next to relaxed that large integvers (> 2^^53) will change if you use relaxed.

Reply:
Good call, added it to the description and now we show a warning banner when relaxed is chosen.